### DISCUSSION TITLE
how to use convert.py

### DISCUSSION THREAD

**Discussion by Gee1111** [ggml-org/llama.cpp#4426] [General]
I wanna convert "flozi00/Mistral-7B-german-assistant-v5" with 3 safetensors files to GGUF ?
What is the command line for that?

**Comment by sussyboiiii** [comment]
Just use -h and then you'll see more. Normally you will only need to specify the path e.g. python3 convert.py /path/to/your/model and then you'll get f16/f32 and then you can use ./quantize to convert to gguf
